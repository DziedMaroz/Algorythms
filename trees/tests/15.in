15
4
3
2
1
10
11
5

