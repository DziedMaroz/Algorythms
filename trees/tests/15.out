15
5
3
2
1
10
11
